source/localizable/_code-of-conduct.en.md

```diff
@@ -1,39 +1,39 @@
 Purpose
 -------
 
-A primary goal of all the conferences and user groups that refer to this Code of Conduct is to be inclusive to the largest number of contributors, with the most varied and diverse backgrounds possible. As such, we are committed to providing a friendly, safe and welcoming environment for all, regardless of gender, sexual orientation, ability, ethnicity, socioeconomic status and religion (or lack thereof).
+A primary goal of all the conferences and user groups that follow this Code of Conduct is to create an inclusive community that welcomes and supports as many people with various backgrounds and identities as possible. As such, we are committed to build a friendly and safer environment for all. These various backgrounds and identities include (but are not limited to) gender identity and expression, sex, sexual orientation, educational level, mental and physical ability, immigration status, race, ethnicity, national origin, social and economic status, age, size and religion (or lack thereof).
```

Comment:
Yes, that"s definitely why we should get feedback from local organization that have more experience in this area :)

another reason why we changed the wording was that "contributors" is difficult to define - newcomers might not feel like they are contributing anything yet. but they are exactly the people to whom we want to be especially inviting :)

and i do believe that it makes sense to explain a bit more extensive who & what we mean rather than, as thuy mentioned, a standard sentence that many might just overread.